packages/app/package.json

```diff
@@ -17,7 +17,8 @@
     "socket.io-client": "2.3.1",
     "tailwindcss": "1.9.6",
     "vue": "3.0.2",
-    "vue-router": "4.0.0-rc.3"
+    "vue-router": "4.0.0-rc.3",
+    "moment": "2.29.1"
```

Comment:
Moment is more or less outdated ([check here](https://sebastiandedeyne.com/moment-js-thank-you-for-your-service/)) so I would suggest using [dayjs](https://github.com/iamkun/dayjs)

Reply:
fixed in c9562eec904cabada1f650c950ed9ae88aacd317